src/stratis_cli/_actions/_physical.py

```diff
@@ -66,12 +66,13 @@ def list_pool(namespace):
             managed_objects,
             props={"Pool": parent_pool_object_path},
         )]
+        sorteddevs = sorted(modevs, key=lambda dev: str.lower(dev.Devnode()))
         tables = [
             [
-                modev.Devnode(),
-                str(Range(modev.TotalPhysicalSize(), SECTOR_SIZE)),
-                state_val_to_string(modev.State()),
-            ] for modev in modevs
+                sorteddev.Devnode(),
+                str(Range(sorteddev.TotalPhysicalSize(), SECTOR_SIZE)),
+                state_val_to_string(sorteddev.State()),
+            ] for sorteddev in sorteddevs
```

Comment:
The effect is much the same right now, but it seems better to make the table and then sort. It's easy to imagine wanting to sort on multiple fields of the table in some later version. So, revert to making tables as in original,
and then:

```
print_table(
     ...,
     [e for e in sorted(tables, key=lambda entry: entry[0]],
     ...,
)
```